conf/messages

```diff
@@ -531,9 +531,10 @@ pages.businessContactDetails.website.label                   = Website address (
 validation.businessContactDetails.email.missing              = Enter an email address
 validation.businessContactDetails.email.invalid              = Enter a valid email address
 validation.businessContactDetails.email.maxlen               = Email address must not exceed 70 characters
-validation.businessContactDetails.mobile.invalid             = Enter a valid contact number
-validation.businessContactDetails.daytimePhone.invalid       = Enter a valid contact number
 validation.businessContactDetails.atLeastOneNumber.missing   = Enter at least one contact number
+errors.invalid.businessContactNum                            = Enter a valid contact number
```

Comment:
is there any reason why it has changed from validation.... to errors,? validation is used elsewhere